yw_website/yw_website/apps/website/templates/extendable/_base_my_workflow_listview.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'extendable/__base_template.html' %}
+
+{% block inherits_from_base_scripts %}
+    {% load static %}
+    {% get_static_prefix as STATIC_PREFIX %}
+    <script src="{{ STATIC_PREFIX }}scripts/viz.js"></script>
+    <script src="{{ STATIC_PREFIX }}scripts/d3.min.js"></script>
+    <script src="{{ STATIC_PREFIX }}scripts/d3-graphviz.min.js"></script>
+{% endblock %}
+
+{% block main_content %}
+
+    <div class="dns">
+        {{host}}
+        <span class="tooltiptext"> Use this as the server address for the YesWorkflow config file</span>
+    </div>
+
+    <div class="list">
+
+        <!-- Content to workflow list_view must be assigned to the document_list keyword argument -->
```

Comment:
Instead of adding this file, This should just override the main_content block in the my_workflow_list template because there is only one small change